code/lib/cli/rendererAssets/nextjs/js/Button.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './button.css';
+
+/**
+ * Primary UI component for user interaction
+ */
+export const Button = ({ primary, backgroundColor, size, label, ...props }) => {
+  const mode = primary ? 'storybook-button--primary' : 'storybook-button--secondary';
+  return (
+    <button
+      type="button"
+      className={['storybook-button', `storybook-button--${size}`, mode].join(' ')}
+      {...props}
+    >
+      {label}
+      <style jsx>{`
```

Comment:
This is exactly like the React version of this template file, with the inline style swapped for styled-jsx, Next.js' preferred (CSS-in-JS) styling solution.

Reply:
@kylegach is it guaranteed that a user will have `styled-jsx` installed if they are using `next`? I don't think we do in our monorepo... 🤔

Reply:
It worked as expected when running the Storybook generated by the `yarn generate-repros-next --template nextjs/default-js --local-registry` command, so I think the answer is "yes".

Reply:
`styled-jsx` comes out of the box with Nextjs:
https://nextjs.org/blog/styling-next-with-styled-jsx